public/player.js

```diff
@@ -35,32 +35,42 @@
     YT.ready(onYouTubeIframeAPIReady);
   };
 
-  function toggleHighlight(x) {
-    x.classList.toggle("highlighted-option")
-    setTimeout(function() {
-      x.classList.remove("highlighted-option")
-    }, 3000)
+  var funcCount = (function () {
+      var counter = 0;
+      return function () {return counter += 1;}
+  })();
+
+  function toggleHighlight(answer, arr, event) {
+      for (var i=1; i<arr.length; i++) {
+        if (arr[i].classList.contains("highlighted-option") === true) {
```

Comment:
`if (x === true) x = false`
is basically the same as 
`x = false`

can remove the if and just write `arr[i].classList.remove("highlighted-option")`